addons/kotsadm/alpha/install.sh

```diff
@@ -107,6 +109,11 @@ function kotsadm() {
     kubectl label pvc kotsadm-postgres-kotsadm-postgres-0 velero.io/exclude-from-backup- kots.io/backup=velero --overwrite
 
     kotsadm_cli $src
+
+    # Migrate existing hostpath and nfs snapshot minio to 
+    if [ "$KOTSADM_DISABLE_S3" == "1" ]; then
+        kubectl kots velero migrate-minio-filesystems -n default
```

Comment:
have we thought about all the scenarios that this command might affect? what if the velero addon is not included for example? i see that the command will fail if it didn't detect velero https://github.com/replicatedhq/kots/blob/b61a9a23f7d746dc8eaeabdd90fbeff2170e23dc/pkg/kotsadm/minio.go#L142

Reply:
what if an object store isn't used?

Reply:
i know that customers need to explicitly pass this flag now, but it will be opt-out in the near future